src/extension/background-script/index.js

```diff
@@ -83,7 +83,7 @@ const routeCalls = (message, sender) => {
   }
   const debug = state.getState().settings.debug;
 
-  const action = message.type; //|| message.action; // TODO: what is a good message format to route to an action?
+  const action = message.action; //|| message.action; // TODO: what is a good message format to route to an action?
```

Comment:
Because this is a tricky change and if we missed a place where we used `type` the app will break. 

I suggest to do something like this here: 

```js
if (message.type) { 
  console.debug("Invalid message, using type", message);
}
const action = message.action || message.type;
```

this would give us a log entry and would not break the app. 
what do you think?

Reply:
That should help as we have a limited number of unit tests!

Reply:
Done!